pkg/doc/did/doc.go

```diff
@@ -790,6 +795,16 @@ func (r *didKeyResolver) Resolve(id string) (*verifier.PublicKey, error) {
 	return nil, ErrKeyNotFound
 }
 
+// MapKeyTypeToKMS maps key type of DID doc to the KMS one.
+func MapKeyTypeToKMS(t string) (kms.KeyType, error) {
+	switch t {
+	case "Ed25519VerificationKey2018":
+		return kms.Ed25519Type, nil
```

Comment:
I'm a bit confused. What's the purpose of using interface{} above when we are mapping to definitions in kms anyways?